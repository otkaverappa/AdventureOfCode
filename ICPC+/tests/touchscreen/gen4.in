19
uwtf 10
ksjm
rivv
rytm
bfds
qyte
szax
vxar
uvwg
pfsh
poei
ozup 10
ezkw
yfas
gaqb
yssu
qoah
vvom
juvx
trpa
rzwr
gzlm
zdoz 10
wivo
xyvs
tmfe
gcdc
tscm
tadz
zpob
uccs
kagj
ycet
qjzw 10
ncyg
xcts
fyue
pihk
mkcy
kjik
nofd
xebm
icsh
glal
kusz 10
ezls
loqx
xahm
ompn
sszb
usid
fkor
ggtk
gect
svrr
xadm 10
nubh
pbkl
vtoc
ddwm
jpwr
vbnq
yehv
enjt
hlbw
olil
gwnm 10
blyn
awgy
ztox
xvve
iexs
paqf
oyqw
xgiy
rilu
fusg
nign 10
fbrq
fqkx
rach
bvdy
bmyu
wmod
gikt
sqgy
rxqy
qavh
aaqb 10
xtba
haue
mljs
vtnq
juoc
tgbj
gysi
aikz
elzn
nwtz
hdte 10
whui
dkky
qnky
ochq
ntqt
gpgu
lavu
fpzb
yvld
gyby
nlzb 10
qgsd
bkyh
bebn
gzjl
qkpo
gato
auoo
inpy
tjdu
vdex
hhmq 10
gvdw
gsmo
vfev
csjm
gbmz
mrwi
ucfd
jrvq
ozow
rbmo
iqkm 10
jvyp
wmql
dotx
qadc
tysh
xigr
jshu
jtgu
phjn
tbyz
styi 10
tbmm
cewz
ncsy
wauf
tdbk
mmah
oagi
ufsp
ifck
jamy
egza 10
ivie
yjom
wqum
raul
hmcs
reed
gqdk
zelh
atla
ecna
ujol 10
jiyt
xbno
hrto
kxyj
blqd
geel
itnc
ccqn
koih
rvyz
ntpx 10
qpgu
byxh
eduo
yita
kjow
zwgt
sgsh
zjeq
ymmz
kljr
rffp 10
nyqx
jgwl
ccew
iwdh
fjzf
vlhi
zszq
xefl
fxkq
dgbi
kfes 10
dkak
vbqr
pxbo
pago
gnzn
lmeo
sfyf
nfxq
rzbm
btfs
