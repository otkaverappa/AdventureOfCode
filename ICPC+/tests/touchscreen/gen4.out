uvwg 8
ksjm 10
rytm 11
bfds 12
pfsh 12
qyte 12
rivv 13
vxar 13
poei 17
szax 17
gzlm 13
yssu 14
ezkw 16
rzwr 16
vvom 17
gaqb 19
trpa 22
yfas 23
juvx 26
qoah 30
zpob 12
xyvs 14
gcdc 15
tadz 15
uccs 19
ycet 20
tmfe 21
tscm 21
kagj 22
wivo 26
fyue 15
mkcy 15
xcts 15
glal 16
nofd 16
icsh 19
xebm 19
ncyg 23
kjik 24
pihk 24
loqx 6
gect 15
ggtk 18
ompn 18
sszb 18
usid 18
fkor 19
svrr 19
ezls 23
xahm 24
ddwm 6
yehv 15
nubh 16
enjt 17
vtoc 18
vbnq 19
hlbw 23
jpwr 23
pbkl 23
olil 26
awgy 9
blyn 12
fusg 14
rilu 14
iexs 15
xgiy 15
xvve 16
ztox 18
paqf 19
oyqw 23
gikt 8
bmyu 9
bvdy 11
sqgy 14
rach 16
fbrq 17
fqkx 17
qavh 18
wmod 18
rxqy 19
elzn 14
haue 15
nwtz 16
gysi 17
tgbj 18
xtba 18
aikz 20
vtnq 22
juoc 23
mljs 26
ochq 9
yvld 9
gyby 10
ntqt 10
lavu 12
gpgu 14
dkky 15
whui 15
qnky 17
fpzb 20
bkyh 11
tjdu 12
bebn 13
vdex 15
qgsd 16
gato 22
inpy 22
gzjl 23
auoo 25
qkpo 25
gbmz 5
jrvq 7
gvdw 10
gsmo 13
rbmo 13
ucfd 13
ozow 15
vfev 16
csjm 17
mrwi 19
jshu 8
phjn 10
jtgu 11
jvyp 15
tysh 16
qadc 17
tbyz 18
dotx 23
xigr 23
wmql 25
tdbk 11
tbmm 12
wauf 12
ifck 15
jamy 15
oagi 15
ufsp 15
ncsy 16
cewz 17
mmah 19
ecna 8
hmcs 10
reed 10
atla 13
gqdk 18
ivie 19
raul 21
wqum 21
zelh 21
yjom 22
koih 9
hrto 11
jiyt 11
kxyj 13
geel 14
itnc 16
xbno 16
rvyz 19
blqd 20
ccqn 23
kjow 9
yita 12
ymmz 12
kljr 16
byxh 17
zjeq 18
zwgt 19
eduo 20
sgsh 20
qpgu 25
dgbi 7
xefl 8
jgwl 10
vlhi 11
iwdh 13
ccew 15
fjzf 15
fxkq 17
zszq 20
nyqx 21
btfs 8
nfxq 8
sfyf 11
vbqr 12
lmeo 13
gnzn 15
pago 17
dkak 18
pxbo 18
rzbm 19
